src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,58 @@
+
+This BEP has been initiated by members of the INCF Working Group on Standardized Data Structures,
+that was initiated in 2020 to develop a set of specifications and tools
+that would allow the standardization of a directory structure for experimental data recorded
+with animal models in neuroscience, and its associated metadata.
+
+Please consider joining this working group if you would like to contribute to this effort.
+You can also reach the moderators of this BEP through our [main discussion forum](https://github.com/INCF/neuroscience-data-structure/issues), where you can participate in existing discussions or raise new questions / issues.
+
+Most core principles of the original BIDS and particulars of BIDS-iEEG specification are adopted
+for this modality as well, though some special considerations and additional fields were added.
+
+Several [example Microelectrode Electrophysiology datasets](https://bids-standard.github.io/bids-examples/#microephys)
+have been formatted using this specification and can be used for practical guidance when curating a new dataset.
+
+## Primary data file formats
+
+Unprocessed microelectrode electrophysiology (`microephys`) modality data (of `icephys` or `ecephys` datatypes) must be stored in an [open file format](https://en.wikipedia.org/wiki/Open_format),
+while the native format, if different, can be stored in an optional  `sourcedata/` directory.
+The native file format is used in case conversion elicits the loss of crucial metadata specific to manufacturers and specific acquisition systems.
+Metadata should be included alongside the data in the `.json` and `.tsv` files.
+The current list of allowed data file formats:
+
+| **Format**                                                                          | **Extension(s)** | **Description**                                                                                                                                                                                                      |
+--------------------------------------------------------------------------------------|------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Neuroscience Information Exchange Format](https://nixio.readthedocs.io/en/latest/) | `.nix`           | A generic and open  framework with an hdf5 backend and a defined interface to many ephys formats via the [Neo library](https://neo.readthedocs.io/en/latest/). The `.nix` file has to contain a valid Neo structure. |
+| [Neurodata Without Borders](https://www.nwb.org)                                    | `.nwb`           | BRAIN Initiative Data Standard based on an hdf5 backend ...                                                                                                                                                          |
```

Comment:
Is the of the ellipse that you are looking for someone to fill in the details?

Reply:
yes, ... I will copy paste from the google doc which says now

> An open data standard for neurophysiology, including data from intracellular and extracellular electrophysiology experiments.